### ISSUE TITLE
Missing dependency for 'sqlalchemy.orm.dependency'

### ISSUE THREAD

**Issue by Bingdom** [Nuitka/Nuitka#2262]
Here's an example:

```py
# nuitka-project: --standalone
# gcc doesn't work. Use clang
# nuitka-project: --clang

from sqlalchemy.orm import DeclarativeBase
from sqlalchemy.ext.asyncio import AsyncAttrs

class Base(AsyncAttrs, DeclarativeBase):
    pass

print("Hello")
```

Compile it, and you will get the following error after running:
```
Traceback (most recent call last):
  File "/home/james/test.dist/test.py", line 5, in <module>
  File "/home/james/test.dist/sqlalchemy/orm/__init__.py", line 171, in <module sqlalchemy.orm>
  File "/home/james/test.dist/sqlalchemy/orm/__init__.py", line 167, in __go
  File "/home/james/test.dist/sqlalchemy/util/preloaded.py", line 139, in import_prefix
ModuleNotFoundError: No module named 'sqlalchemy.orm.dependency'
```

This works perfectly fine without compiling, and will output "Hello"

  > python -m nuitka --version

```
1.5.8
Commercial: 2.0.0
Python: 3.10.10 (main, Mar  9 2023, 04:12:18) [GCC 10.2.1 20210110]
Flavor: Unknown
Executable: /usr/local/bin/python3.10
OS: Linux
Arch: aarch64
Distribution: Debian 11
Version C compiler: /usr/bin/gcc (gcc).
```

> How did you install Nuitka and Python

Compiled python myself. Installed nuitka via:
``python3.10 -m pip install -U --force-reinstall git+ssh://git@github.com/Nuitka/Nuitka-commercial.git``

  > python -m pip freeze
```
SQLAlchemy==2.0.15
```

**Comment by Bingdom** [comment]
Like #2208, I just had to explicitly import the module to fix it.
``--include-module=sqlalchemy.orm.dependency``

**Comment by kayhayen** [comment]
It still would be nice to have these automatic.

If you are available to do a PR, this can be fixed by adding to the yaml configuration, described here: [https://nuitka.net/doc/nuitka-package-config.html](https://nuitka.net/doc/nuitka-package-config.html)

And here is more of a walk through style post: [https://nuitka.net/posts/nuitka-package-config-kickoff.html](https://nuitka.net/posts/nuitka-package-config-kickoff.html)

Here are a example PRs for other packages:

[#2030](https://github.com/Nuitka/Nuitka/pull/2030) 
[#1993](https://github.com/Nuitka/Nuitka/pull/1993)

**Comment by kayhayen** [comment]
Thanks for working on this one, very good.

**Comment by kayhayen** [comment]
This is merged and will be in 1.7, as there won't be a hotfix for 1.6 anymore at this point.

**Comment by kayhayen** [comment]
Part of the 1.7 release I just made.
